src/gridcoin/upgrade.cpp

```diff
@@ -177,6 +185,18 @@ void Upgrade::SnapshotMain()
     std::cout << _("Snapshot Process Has Begun.") << std::endl;
     std::cout << _("Warning: Ending this process after Stage 2 will result in syncing from 0 or an incomplete/corrupted blockchain.") << std::endl << std::endl;
 
+    // Verify a mandatory release is not available before we continue to snapshot download.
+    std::string VersionResponse = "";
+
+    if (CheckForLatestUpdate(VersionResponse, false, true))
+    {
+        std::cout << _("Unable to perform a Snapshot download as the wallet has detected that a new mandatory version is available for download.") << std::endl;
```

Comment:
should i add that part to the one above as well in updateqt

Reply:
i added to both but i reworded to "before the snapshot can be downloaded and applied." i believe it sounds better but if you decide otherwise after next commit i will change